Makefile

```diff
@@ -38,6 +38,7 @@ test-data:
 	cp ${gitDataDir}rac-experiments-v3.json ${testDataDir}
 	cp -r ${gitDataDir}assignment-v2 ${testDataDir}
 	rm -rf ${tempDir}
+	yarn obfuscated-mock-rac
```

Comment:
To replace with `cp ${gitDataDir}rac-experiments-v3-obfuscated.json ${testDataDir}` when https://github.com/Eppo-exp/sdk-test-data/pull/2 get approved

We don't want to have an equivalent of `test/writeObfuscatedMockRac.ts` in multiple languages, so I'm thinking we'll have code to generate the mock rac file here but tests should source the file from sdk-test-data

Reply:
Replaced!